### ISSUE TITLE
Add support for character encoding in Basic Auth

### ISSUE THREAD

**Issue by sify21** [a85/POSTMan-Chrome-Extension#4070]
## App Details:
```
Postman for Linux
Version 5.5.0
linux 3.10.0-693.11.1.el7.x86_64 / x64
```
## Issue Report:
 1. Using Basic Auth, When username has chinese characters, the encoded authorization header is not correct. This bug always been there.
 2. Expected behaviour: 
postman should encode the auth string with utf-8, then with base64

## Steps to reproduce the problem
1. Prepare a web application.
I'm using a spring web application. A GET method is secured by spring-security using basic-auth. Create a user in the application: username=中文, password=123abc
2. In postman, choose Basic Auth with above username and password, then send a GET request to that api. 
3. The result:
A header is added by postman: "Authorization: Basic LYc6MTIzYWJj", and the result is 401 unauthorized. The auth string "LYc6MTIzYWJj" is not right, which decodes to "-:123abc". The correct string should be "5Lit5paHOjEyM2FiYw==", which decodes to "中文:123abc". if I manually change the auth string to the correct one, the request would work.

**Comment by harryi3t** [comment]
@sify21 Thanks for reporting this bug. We will look into this.
Will update the thread once the bug is confirmed.

**Comment by null9beta** [comment]
hi @harryi3t 

i can report the same issue here. Actually i could break it down into the same encoding issue.

Postman uses ISO-8859-15 encoding the basic auth header but probably should use UTF-8 doing so.
In our case our golang backend fails because decoding is done using UTF-8 as well.

A potential solution not to introduce a BC might be using the settings to be able to choose the charset for encoding the auth header.

As a workaround we are creating the Authorization Header manually putting it explicitly into the headers list. That a least solves the problem.

Maybe that information helps, looking forward seeing this getting into the product.

And thanks for an awesome tool :)

**Comment by harryi3t** [comment]
@null9beta Thanks for your input. It made me look back the encoding we are using. We have identified the issue and are fixing it.
Will update the thread once it's ready to be released.

**Comment by slawa-dev** [comment]
@harryi3t Just run into this issue as well. Have to create Basic Authorization header manually if the username or password has umlauts.

**Comment by shamasis** [comment]
Comments from @null9beta and @Lysander at https://github.com/postmanlabs/postman-app-support/issues/5161#issuecomment-479814001 seems to be the most optimum route at this moment. Quoting it here:

> Overall it's not about unicode but about the encoding that is used for generating the basic auth credentials.
> 
> Currently postman uses ISO-8859-1 as encoding; in #4070 there is the suggestion to change it to UTF-8. Imho that would not be a good solution!
> 
> I would apreciate it, if one could choose the encoding. This would be the best solution reagrding compatibility with older and newer webservices.

**Comment by vikiCoder** [comment]
Hi everyone,

This issue is fixed in the latest release of Postman App (v7.14.0). We now use UTF-8 encoding while calculating `Authorization` header for basic auth as suggested in the [Basic auth RFC](https://tools.ietf.org/html/rfc7617#section-2.1). Please update your Postman app to the latest version and let us know if you are still facing any issues regarding this.
